js/util.ts

```diff
@@ -122,6 +122,17 @@ export function isObject(o: unknown): o is object {
   return o != null && typeof o === "object";
 }
 
+// Returns whether o is iterable.
+// @internal
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isIterable(obj: any): obj is Iterable<any> {
```

Comment:
unknown instead of any?

I think `@internal` is not needed here as this file isn't exported from `js/globals.ts`

Reply:
generics is a better choice